examples/website-demos/social-messaging/src/assets/GiphyIcon.tsx

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
 
 export const GiphyIcon: React.FC = () => (
-  <svg width='70' height='24' viewBox='0 0 70 24' fill='none' xmlns='http://www.w3.org/2000/svg'>
+  <svg
+    width='70'
+    height='24'
+    viewBox='0 0 70 24'
+    fill='none'
+    xmlns='http://www.w3.org/2000/svg'
+    style={{ flexShrink: 0 }}
```

Comment:
why is this style needed?

Reply:
When the channelContainer shrinks to accommodate the thread, the GiphyIcon shrinks as well without this style